tests/misc/msizeof5.nim

```diff
@@ -116,7 +116,7 @@ when defined caseBad:
     ensureCgen Foo5
 
   block:
-    type Foo6{.importc, completeStruct.} = object
+    type Foo6{.importc:"struct Foo6", completeStruct.} = object
```

Comment:
Why are you changing the test?

Reply:
so that it works with `nim cpp`; test was never run before (refs https://github.com/nim-lang/Nim/issues/14378);
I wrote these tests btw